README.md

```diff
@@ -1,5 +1,8 @@
 # konk
 
+
+Office Hour 11am CST on Fridays landing on even days [https://infoblox.zoom.us/j/98130157567](https://infoblox.zoom.us/j/98130157567)
```

Comment:
I'm not a an of the placement. I feel like this doesn't need to be the very first thing in the readme. Maybe put it after the description.

At a minimum, we should advertise in UTC. CST is a bit obscure. I wonder if there's a tool we could link that would place this time on a chart showing local times...

Reply:
https://www.worldtimebuddy.com/?qm=1&lid=100,30,8,6&h=100&date=2020-11-19&sln=17-17.5&hf=0

Reply:
how about tv time, 8AM PST/12 EST

Reply:
Good enough.